Content.Shared/Weapons/Melee/MeleeWeaponComponent.cs

```diff
@@ -21,6 +21,18 @@ public sealed partial class MeleeWeaponComponent : Component
     [DataField, ViewVariables(VVAccess.ReadWrite)]
     public bool AltDisarm = true;
 
+    /// <summary>
+    /// If you can't light attack
+    /// </summary>
+    [DataField, ViewVariables(VVAccess.ReadOnly)]
```

Comment:
datafields are automatically vv ro, make it readwrite or remove the viewvariable attribute

Reply:
don't enough know why it's RO, will do